test/static_a_token.test.ts

```diff
@@ -1,8 +1,10 @@
 import { StarknetContract, Account } from "hardhat/types";
 import { starknet } from "hardhat";
+import { BigNumber } from "ethers";
 import { TIMEOUT, L1_TEST_ADDRESS } from "./constants";
 import { expect } from "chai";
-import { wadToRay, decimalToWad } from "./rayMath";
+import "./wadraymath";
+import { WAD } from "./wadraymath";
```

Comment:
merge these two lines, we prefer "targeted imports" i.e. listing the variables/functions you'd like to import